packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
Aaron,

thanks for your time. I took the code from startup, which works without errors. But it makes no difference, if I remove the complete code lines, which handle the locale stuff. 

```
                //data.language = data.language.split('-').shift().toLowerCase();
                //if (data.language !== 'en') {
                //      Meteor.call('loadLocale', data.language, (err, localeFn) => {
                //              Function(localeFn).call({moment});
                //              moment.locale(data.language);
                //      });
                //}
```

Ciao
Marcus